DataHolderLib/DataExplorerProject.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef DATAEXPLORERPROJECT_H_
+#define DATAEXPLORERPROJECT_H_
+
+#include <memory>
+
+#include "GeoLib/GEOObjects.h"
+
+namespace MeshLib {
+	class Mesh;
+}
+
+/**
+ * The ProjectData Object contains all the data needed for a certain project, i.e. all
+ * geometric data (stored in a GEOObjects-object), all the meshes, processes,
+ * and process variables.
+ */
+class DataExplorerProject final
```

Comment:
For at least consistency reasons I would move the class in a namespace `DataHolderLib`.